models/sdn-fabric-0.1.x/sdn-fabric-0.1.0.tree

```diff
@@ -0,0 +1,36 @@
+module: onf-switch
+  +--rw switch* [switch-id]
+     +--rw switch-id       yt:yang-identifier
+     +--rw display-name?   string
+     +--rw description?    string
+     +--rw model?          -> /sm:switch-model/switch-model-id
+     +--rw role            enumeration
+     +--rw management
+     |  +--rw address?       inet:ipv4-address
+     |  +--rw port-number?   inet:port-number
+     +--rw port* [port-id]
+     |  +--rw port-id         -> /sm:switch-model/port/port-id
+     |  +--rw display-name?   string
+     |  +--rw description?    string
+     |  +--rw speeds*         identityref
+     |  +--rw channel?        enumeration
```

Comment:
If the port ID uses a syntax with slashes (for example 1/2), is the channel already implicit in the ID ?